ray-operator/controllers/ray/common/job.go

```diff
@@ -112,6 +113,16 @@ func GetDefaultSubmitterTemplate(rayJobInstance *rayv1alpha1.RayJob) v1.PodTempl
 				{
 					Name:  "ray-job-submitter",
 					Image: image,
+					Resources: v1.ResourceRequirements{
+						Limits: v1.ResourceList{
+							v1.ResourceCPU:    resource.MustParse("1"),
```

Comment:
Why do we select 1 CPU and 1 GB of memory rather than other values?

Reply:
It's a guess. Let me try to benchmark it

Reply:
Updated using the benchmark result.